pyproject.toml

```diff
@@ -22,31 +22,36 @@ PyJWT = "1.7.1"
 PyPDF2 = "1.26.0"
 PyYAML = "4.2b4"
 SOAPpy = "^0.12.22"
+Sphinx = ">=2.4.0"
 Voicent-Python = "^1.0"
 Werkzeug = "0.11.15"
 XlsxWriter = "0.9.3"
 agithub = "^2.2.2"
 altair = "^4.1.0"
+argcomplete = "^1.12.1"
 bokeh = "1.1.0"
 boto = "^2.49.0"
 boto3 = "1.9.102"
+cachetools = ">=2.0.1"
 cerberus = "^1.3.2"
 chardet = "3.0.4"
+colorama = "^0.4.3"
 cryptography = "2.6.1"
 cython = "^0.29.21"
 ddt = "1.2.0"
 decorator = "4.0.10"
 docutils = "0.12"
 ebaysdk = "2.1.5"
 email_validator = "^1.1.1"
-extract-msg = "^0.25.3"
-extract_msg = "^0.25.3"
+escpos = "^1.6"
+extract-msg = "^0.27.4"
```

Comment:
This lib appears 2 times. I think we can remove one occurrence.

Reply:
The real name is extract-msg
https://pypi.org/project/extract-msg/
extract_msg is wrong, the error is in OCA_social module mail_drop_target in __manifest__.py, but correct in requirements.txt
The __manifest__py use the import name.
Strangely, no error because it works without issue with pip.

We need to fix the poetry generator to improve when using dependencies from __manifest__.py.
#15 
Report this issue, because it's not related to the actual functionality.